### PR TITLE
Fix vol flip axis 691

### LINKED ISSUE #691: incorrect Volume.flip default axis
Developers found Volume.flip axis is 0, but this corresponds to the stack axis.

A default of axis=1 probably makes more sense.

### PR DESCRIPTION
**Pull request by garrettwrong**
Closes #691

### CODE REVIEW

**Comment by chris-langfield** [comment]
I wonder if it should block you from doing `axis=0` to prevent confusion?

**Comment by garrettwrong** [comment]
> I wonder if it should block you from doing `axis=0` to prevent confusion?

Yeah I thought about that too, but it involved more testing.  I'll bite.  Give me a few minutes to change the tests. Thanks.

**Comment by garrettwrong** [comment]
Weird, some tests I would have thought unrelated are failing.  Let me see if they are using `flip` (oof...).

**Comment by garrettwrong** [comment]
Yeah, we were in fact using an `axis=0` flip to reorder some eigenvolumes (intentionally).

File: src/aspire/source/simulation.py [hunk 1]
```diff
@@ -265,7 +265,7 @@ def eigs(self):
 
         # Arrange in descending order (flip column order in eigenvector matrix)
         w = w[::-1]
-        eigs_true = eigs_true.flip()
+        eigs_true = Volume(eigs_true[::-1])
```
Comment (chris-langfield) [inline src/aspire/source/simulation.py:268]: This is cool I always thought you had to do `[:,:,...]`

**Review by chris-langfield — APPROVED** [review]
LGTM thanks!

**Comment by garrettwrong** [comment]
Thanks for the review Chris.  This is minor so I think we can skip Joakim on it. Merging.

**Pull request merged**